src/checker.py

```diff
@@ -283,7 +284,12 @@ def _update_appdata(self):
                 timestamp = datetime.datetime.now()
             else:
                 timestamp = last_update.timestamp
-            add_release_to_file(appdata, last_update.version, timestamp.strftime("%F"))
+            try:
+                add_release_to_file(
+                    appdata, last_update.version, timestamp.strftime("%F")
+                )
+            except SAXParseException as err:
+                log.error(str(err))
```

Comment:
Can we report this out to update_manifests() and its caller, so that the problem can be noted in the commit message and hence PR?

Reply:
This would require some new mechanism to collect warnings and pass them all the way back to `main.py`. I'd prefer to implement this in another PR, since such mechanism could be useful for other things.

Reply:
OK